android/app/src/main/java/org/fossasia/openevent/activities/TrackSessionsActivity.java

```diff
@@ -87,13 +87,13 @@ protected void onResume() {
         //setting the grid layout to cut-off white space in tablet view
         DisplayMetrics displayMetrics = this.getResources().getDisplayMetrics();
         float width = displayMetrics.widthPixels / displayMetrics.density;
-        int spanCount = (int) (width/250.00);
+        int spanCount = (int) (width / 250.00);
```

Comment:
@iamareebjamal Thanks for clarifying, I will remember not to to this from next time.